pages/index.tsx

```diff
@@ -1,35 +1,35 @@
-import Head from "next/head";
-import Image from "next/image";
-import { wrapper } from "@/src/store";
-import serverApi, { useGetAllUsersQuery } from "../src/services/serverApi";
+import Head from 'next/head'
+import Image from 'next/image'
+import { wrapper } from '@/src/store'
+import serverApi, { useGetAllUsersQuery } from '../src/services/serverApi'
 // import NewMembersCard from "@/src/components/NewMember";
-import { NEW_USER, NUM_MEMBERS_NUMBER } from "@/src/constants/AppConstants";
-import styles from "@/styles/Home.module.css";
-import MembersSectionMain from "@/src/components/MembersSectionNew/MembersSectionMain";
-import NewMemberSection from "@/src/components/NewMemberSection";
-import { UserType } from "../src/components/MembersSectionNew/types/MembersSection.type";
+import { NEW_USER, NUM_MEMBERS_NUMBER } from '@/src/constants/AppConstants'
+import styles from '@/styles/Home.module.css'
+import MembersSectionMain from '@/src/components/MembersSectionNew/MembersSectionMain'
+import NewMemberSection from '@/src/components/NewMemberSection'
+import { UserType } from '../src/components/MembersSectionNew/types/MembersSection.type'
 type PictureType = {
-  publicId: string;
-  url: string;
-};
+  publicId: string
+  url: string
+}
```

Comment:
isn't without semicolons this will give error?

Reply:
No

Reply:
I would advise to add rules to add semi colons wherever it's not present, I know in js/ts at most of the places semi-colons are not needed and it just works fine but there are few exceptions wherever it's necessary to use that semi colon so that js/rs doesn't get confused b/w the instructions.

Reply:
Added